src/invocationPanel.ts

```diff
@@ -212,11 +212,19 @@ export class InvocationPanel {
                             const sb = neon.default.create.scriptBuilder();
                             let script = '';
                             if (method.name === 'Main') {
-                                const args = InvocationPanel.parseArrayArgument(method.parameters[1].value);
-                                script = sb.emitAppCall(
-                                    contractHash, 
-                                    method.parameters[0].value, 
-                                    args).str;
+                                if (method.parameters.length === 0) {
+                                    script = sb.emitAppCall(contractHash).str;
+                                    script = script.substr(2); // remove the unnecessary PUSH0 instruction (that leaves a 0 on the call stack after execution)
```

Comment:
should we be filing a bug on neon library for this?

Reply:
Opened https://github.com/CityOfZion/neon-js/issues/502